Core/HLE/sceKernelModule.cpp

```diff
@@ -1697,27 +1699,19 @@ static u32 sceKernelLoadModule(const char *name, u32 flags, u32 optionAddr)
 	if (!module) {
 		if (magic == 0x46535000) {
 			ERROR_LOG(LOADER, "Game tried to load an SFO as a module. Go figure? Magic = %08x", magic);
-			return -1;
+			// TODO: What's actually going on here?
```

Comment:
Right but I don't believe that the error response on real firmware is -1.  If it is it's probably at least for a larger class of error than just these magic bytes.

-[Unknown]